flink-runtime/src/test/java/org/apache/flink/runtime/minicluster/MiniClusterConfigurationTest.java

```diff
@@ -74,4 +77,12 @@ public void testSchedulerType_notOverriddenIfExistingInConfig() {
 			JobManagerOptions.SCHEDULER.defaultValue(),
 			miniClusterConfiguration.getConfiguration().getString(JobManagerOptions.SCHEDULER));
 	}
+
+	@Test
+	public void testDefaultTaskExecutorMemoryConfiguration() {
+		final MiniClusterConfiguration miniClusterConfiguration = new MiniClusterConfiguration.Builder().build();
+
+		assertTrue(TaskExecutorResourceUtils.isManagedMemorySizeExplicitlyConfigured(miniClusterConfiguration.getConfiguration()));
+		assertTrue(TaskExecutorResourceUtils.isShuffleMemoryExplicitlyConfigured(miniClusterConfiguration.getConfiguration()));
```

Comment:
This test confused me first a bit because it seems strange that the memory is explicitly configured as we say we wanna use the defaults. I think it would be easier to understand if we simply assert the exact managed and network memory settings derived from the `Configuration`.

One should first extract the defaults and store them in some static variables from `adjustConfigurationForLocalExecution`.